CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+Notable changes to `alacritty_terminal` crate should be documented in its
+[CHANGELOG](./alacritty_terminal/CHANGELOG.md).
```

Comment:
This is a user-facing document, however this is worded in a developer-centric fashion.

This would probably be better:

```suggestion
Notable changes to the `alacritty_terminal` crate are documented in its
[CHANGELOG](./alacritty_terminal/CHANGELOG.md).
```

Arguably the same could be said about the documentation about the sections, but that is a bit older.